klippy/kinematics/hybrid_corexy.py

```diff
@@ -0,0 +1,106 @@
+# Code for handling the kinematics of hybrid-corexy robots
+# The hybrid-corexy kinematic is also known as Markforged kinematics
```

Comment:
Please use a single line for the file description.  It's preferable for all the code to have the same format file header: 1 line comment, 1 blank line, 1 line copyright, 1 blank line, 1 line license.  You can, of course, add any amount of comment lines after the initial header block.